.github/workflows/main.yml

```diff
@@ -57,14 +57,3 @@ jobs:
       run: docker exec test_container /PrairieLearn/docker/test_js.sh
     - name: Run the Python tests
       run: docker exec test_container /PrairieLearn/docker/test_python.sh
-    - name: Copy code-coverage data out of container
-      run: docker cp test_container:/PrairieLearn/coverage coverage
-    - name: Directory listing for debug
-      run: ls -lR
-    - name: Head of lcov.info for debug
-      run: head -1000 coverage/lcov.info
-    - name: Send code-coverage data to Coveralls
-      uses: coverallsapp/github-action@master
-      with:
-        github-token: ${{ secrets.github_token }}
-      continue-on-error: true
```

Comment:
Why are we getting rid of this?

Reply:
`nyc` mangled stack traces, making it really difficult to debug tests. I've personally almost never looked at code coverage when looking at PRs, so I opted to drop it rather than trying to figure out how to coerce `nyc` into presenting correct stack traces. If you feel strongly about keeping it, I can look into that.

Reply:
nope, just curious

Reply:
One thought actually, it might be interesting just to glance at coverage once just to make a list of all the things that need more tests, e.g. a list of which elements, pages, API endpoints, etc. don't have any tests, or don't have nearly enough.

The we can throw that list in with all the other lists of things to do :joy: .

Reply:
You can check out the current coverage on https://coveralls.io/github/PrairieLearn/PrairieLearn!